### PR TITLE
Remove common protein caps from ligand query

### PR DESCRIPTION
**Pull request by jpattle**
# Description
This change adds names of common protein caps (`NME` and `ACE`) and then excludes any residues (whether they are part of the protein or not) with those names from the query that selects the ligand. This means that proteins that have been capped with these residues will not show the caps as a part of the ligand component created by default. This also affects the `ligandPlusConnected` query similarly.

## Actions

- [X] Added description of changes to the `[Unreleased]` section of `CHANGELOG.md`
- [ ] Updated headers of modified files
- [X] Added my name to `package.json`'s `contributors`

### CODE REVIEW

**Review by arose — CHANGES_REQUESTED** [review]
Thanks @jpattle. I like the change but lets check with @JonStargaryen and @mandarsd if it is OK with them as well.

File: src/mol-plugin-state/helpers/structure-selection-query.ts [hunk 1]
```diff
@@ -350,14 +350,25 @@ const ligand = StructureSelectionQuery('Ligand', MS.struct.modifier.union([
...
+            MS.struct.modifier.union([
+                MS.struct.generator.atomGroups({
+                    'entity-test': MS.core.rel.eq([MS.ammp('entityType'), 'polymer']),
+                    'chain-test': MS.core.rel.eq([MS.ammp('objectPrimitive'), 'atomistic']),
+                    'residue-test': MS.core.set.has([
+                        MS.set(...SetUtils.toArray(PolymerNames)), MS.ammp('label_comp_id')
+                    ])
+                }),
+            ]),
+            MS.struct.modifier.wholeResidues({
```
[the remark this answers is not in the document]
Comment (jpattle) [reply src/mol-plugin-state/helpers/structure-selection-query.ts]: Makes sense, thank you. I have pushed an update

**Comment by jpattle** [comment]
> Thanks @jpattle. I like the change but lets check with @JonStargaryen and @mandarsd if it is OK with them as well.

Absolutely, happy to wait for it to be checked 🙂

**Comment by JonStargaryen** [comment]
Nice change!
And thanks for checking :)

**Comment by jpattle** [comment]
@arose looks like Jon has given the go-ahead, is this good to merge?

**Comment by arose** [comment]
Was waiting for @mandarsd as a PDBe representative.

**Comment by arose** [comment]
Going to merge it. We can always adjust if it is problematic. Thank you @jpattle.

**Pull request merged**